docs/usage/configuration-options.md

```diff
@@ -3194,7 +3194,7 @@ Invalid if used outside of a `packageRule`.
 
 ### matchFileNames
 
-Renovate will compare `matchFileNames` glob matching against the dependency's package file and also lock file if one exists.
+Renovate will compare `matchFileNames` glob or RE2 regex matching against the dependency's package file and also lock file if one exists. A regex pattern must be wrapped in `/`.
```

Comment:
```suggestion
Renovate will compare `matchFileNames` glob or RE2 regex matching against the dependency's package file and also lock file if one exists.
```

Link to the glob or regex docs instead of providing an incomplete description

Reply:
This link: https://docs.renovatebot.com/string-pattern-matching/